lua/telescope/actions/init.lua

```diff
@@ -1,4 +1,4 @@
--- Actions functions that are useful for people creating their own mappings.
+jjjjj-- Actions functions that are useful for people creating their own mappings.
```

Comment:
:laughing:

Reply:
@Conni2461 hhhhhhhhhhhhhhhhh forgot that i deleted stopinsert and that happens :D